app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,43 @@
+class AnswersController < ApplicationController
+  before_action :set_question 
+  before_action :set_answer, only: [:edit,:destroy]
+  def new
+    @answer = @question.answers.build
```

Comment:
do we have a separate page to add answer?

Reply:
no